src/Plugin/Visualisation/Style/VisualisationStyleBase.php

```diff
@@ -365,7 +365,7 @@ public function getSourceRecords() {
    *   The fields.
    */
   protected function fields() {
-    return array_filter($this->config('data', 'fields'));
+    return array_unique(array_filter($this->config('data','fields')));
```

Comment:
please add the space back between 'data', and 'fields'

Reply:
Done